app/src/main/java/com/example/footprnt/Map/Util/MapUtil.java

```diff
@@ -161,6 +173,12 @@ public static void setPostImages(Post post, PostAdapter.ViewHolder holder, Conte
         }
     }
 
+    /**
+     * Parses JSONarray of continents
+     *
+     * @param activity
```

Comment:
If you're not going to add onto the default template, you don't need to include it. It's 100% fine to just have this method be annotated with
/**
* Parses JSONarray of continents
*/
Generally you include the variables in the Javadoc if you want to explain why it's needed or the limitations or error cases.